tests/check_shebang_scripts_are_executable_test.py

```diff
@@ -9,6 +9,62 @@
 from pre_commit_hooks.check_shebang_scripts_are_executable import main
 from pre_commit_hooks.util import cmd_output
 
+skip_win32 = pytest.mark.skipif(
```

Comment:
these should be xfail if anything -- then you shouldn't need the no cover

Reply:
Neat function. I tried removing the no cover and got [a test coverage failure](https://asottile.visualstudio.com/asottile/_build/results?buildId=12893&view=logs&j=c5301fb6-0b07-5e6e-20f2-e78065fa524c&t=60800355-5684-5485-0283-7dd0b5fe8f8f) though. Did I miss something here?